packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/configWithKeyVaultPlugin.result.bicep

```diff
@@ -1,6 +1,8 @@
+// Get existing app settings and site config for merge
```

Comment:
Do we really need to sync up comments changes?

Reply:
The comment is visible to end users to help them better understand the bicep content. So if we didn't update the expected value in test case, the case will fail due to missing these comments.